CocoaPods/iphonesimulator-10.0/module.modulemap

```diff
@@ -0,0 +1,4 @@
+module CSQLite [system] {
+    header "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneSimulator.platform/Developer/SDKs/iPhoneSimulator10.0.sdk/usr/include/sqlite3.h"
```

Comment:
As above, this causes build errors for all users who don't have their copy of Xcode 8.x at that path.